### DISCUSSION TITLE
Change Detection Disabled > ENABLED

### DISCUSSION THREAD

**Discussion by fritzisa** [code4recovery/12-step-meeting-list#912] [Q&A] [answered]
Our Area 59 site: https://area59aa.org uses (13) feeds from our Districts and Intergroups. When I set up our site, I did not use "Change Detection Enabled." How can I most easily update to begin using Change Detection?

Thank you for your service.

**Scott F**.  |  Area 59 web Servant  |  [webservant@area59aa.org](url)

**Comment by kiyote33** [comment, marked as answer]
Hi @fritzisa,
See our [website FAQ](https://code4recovery.org/docs/12-step-meeting-list) "How Can I Enable Change Detection For My Disabled Data Source?"  Let me know if you need anything else. 
If that link doesn't work for you, try this one: [12 Step Meeting List Documentation](https://wordpress.org/plugins/12-step-meeting-list/#How%20can%20I%20enable%20Change%20Detection%20for%20my%20disabled%20data%20source%21)

**Reply by tim-rohrer** [reply]
Hi Wayne. The issue is that users can't _edit_ an existing link to enable/disable change notification. They can only delete the import setting and then recreate it.

**Reply by kiyote33** [reply]
Correct

**Reply by kiyote33** [reply]
FYI: Even for a large area with lots of feeds, it won't take that long to reload them all after dropping them first. It could be done pretty quickly the next time they plan on Refreshing a feed. It's a 1 time thing to reload the feeds. I guess the question is, does the benefits of Change Detection make it worth the effort to drop and reload. I'm thinking that for most it's a no-brainer, as Change Detection tells you exactly when a feed is different from what is stored. Only then do you need to do a Refresh.

If we are discussing making code changes here, I'm thinking it probably wouldn't happen this year as my calendar is full. We can certainly discuss bumping priorities or applying other resources to get it done sooner. Might be an opportunity for a new developer to get their feet wet? Eh!

**Reply by tim-rohrer** [reply]
For the foreseeable future, dropping and recreating seems like the best option.
